website/pages/docs/commands/node/drain.mdx

```diff
@@ -65,8 +67,10 @@ operation is desired.
 - `-no-deadline`: No deadline allows the allocations to drain off the node
   without being force stopped after a certain deadline.
 
-- `-ignore-system`: Ignore system allows the drain to complete without stopping
-  system job allocations. By default system jobs are stopped last.
+- `-ignore-system`: Ignore system allows the drain to complete without
```

Comment:
@tgross I kinda wish this flag was default "on", I've yet had a case where I haven't had to set this to true in my various nomad deployments

Reply:
I think I agree. Let me float that by the rest of the team to see if it's something we feel safe flipping before 1.0.

Reply:
I've opened https://github.com/hashicorp/nomad/issues/8622 for further discussion of that default. Thanks for raising it, @jippi!